src/btf.cpp

```diff
@@ -51,6 +51,10 @@ BTF::BTF(const std::set<std::string> &modules) : state(NODATA)
     btf_objects.push_back(
         BTFObj{ .btf = btf__parse_raw(path), .id = 0, .name = "" });
     vmlinux_btf = btf_objects.back().btf;
+    if (!vmlinux_btf) {
+      LOG(V1) << "BTF: failed to parse BTF";
```

Comment:
nit: I feel like this should be a LOG(WARNING) as it can affect the program behavior.

Reply:
Good catch - not nit IMO. If user explicitly requested this BTF and we cannot parse it, we really should let user know. The non-requested V1 log below is fine IMO. We opportunistically parse BTF in the non-requested case.